core/gallery/models.go

```diff
@@ -165,27 +169,29 @@ func InstallModel(systemState *system.SystemState, nameOverride string, config *
 	}
 
 	// Download files and verify their SHA
-	for i, file := range config.Files {
-		log.Debug().Msgf("Checking %q exists and matches SHA", file.Filename)
+	if downloadStatus != nil {
```

Comment:
is this necessary? download status is optional, no need to gate here

Reply:
Removed.